desc/compute/_core.py

```diff
@@ -78,6 +78,96 @@ def _zeta(params, transforms, profiles, data, **kwargs):
     return data
 
 
+@register_compute_fun(
+    name="theta_sfl",
+    label="\\vartheta",
+    units="rad",
+    units_long="radians",
+    description="PEST straight field line poloidal angular coordinate",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["theta", "lambda"],
+)
+def _theta_sfl(params, transforms, profiles, data, **kwargs):
+    data["theta_sfl"] = data["theta"] + data["lambda"]
+    return data
+
+
+@register_compute_fun(
+    name="alpha",
+    label="\\alpha",
+    units="~",
+    units_long="None",
+    description="Field line label",
+    dim=1,
+    params=[],
+    transforms={},
+    profiles=[],
+    coordinates="rtz",
+    data=["theta_sfl", "zeta", "iota"],
+)
+def _alpha(params, transforms, profiles, data, **kwargs):
+    data["alpha"] = data["theta_sfl"] - data["iota"] * data["zeta"]
```

Comment:
OK I've convinced myself why the equation is correct with the minus sign

Reply:
As an explanation, we want $d\alpha = 0$ along a field line, and we also have $\iota = d\theta/d\zeta$ along a field line, which becomes $d\theta - \iota d\zeta = 0$, so then $\alpha = \theta - \iota \zeta$.

I think it might have been weird before the jacobian sign fix, because in that case there's an extra implicit minus sign on iota

Reply:
Thanks. What about my comment on alpha in different coordinate systems (namely Boozer coordinates)?

Reply:
Shouldn't the field line label be independent of coordinates? As long as we all agree on theta=zeta=0 where alpha=0 I would think that changing coordinates shouldn't change the field line label.

Reply:
$\alpha = \theta - \iota \zeta$

$\theta_B = \theta + \iota \nu$

$\zeta_B = \zeta + \nu$

$\alpha_B = \theta_B - \iota \zeta_B = \theta + \iota \nu - \iota \zeta - \iota \nu = \alpha$